pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -92,9 +92,9 @@ in stdenv.mkDerivation {
     })
   ++ [
     ./qtx11extras.patch
-    # Kernel 5.3 fix, should be fixed with VirtualBox 6.0.14
-    # https://www.virtualbox.org/ticket/18911
-    ./kernel-5.3-fix.patch
+    # Kernel 5.4 fix, should be fixed with next upstream release
+    # https://www.virtualbox.org/ticket/18945
+    ./kernel-5.4-fix.patch
```

Comment:
Can you `fetchpatch` these instead of adding them to the repo?

Reply:
You could literally copy the `fetchpatch` statements from #75463.

Reply:
I never got the build completely through with the patches in that PR so unfortunately it wouldn't be sufficient to copy that PR.

Reply:
But perhaps it would work with copying them to the guest-additions package as well?

Reply:
> You could literally copy the `fetchpatch` statements from #75463.

I could not find the patch URLs before on the virtualbox site. Using the URLs from #75463 is working great.

> I never got the build completely through with the patches in that PR so unfortunately it wouldn't be sufficient to copy that PR.

There is a third cahngeset which was missing in your PR. Adding https://www.virtualbox.org/changeset/81649/vbox builds fine.

> But perhaps it would work with copying them to the guest-additions package as well?

It's not so easy to use fetchpatch for the guest additions. Only some of the changes are needed and they are in multiple sub folders. I'm looking for a solution to reuse the upstream patches.